frontend/src/features/permits/apiManager/permitsAPI.ts

```diff
@@ -119,7 +121,7 @@ export const getApplicationsInProgress = async ({
   applicationsURL.searchParams.set("page", (page + 1).toString());
```

Comment:
Do we really need toString() for applicationsURL.searchParams.set()? If it only accepts a string, then try to use string interpolation:
```
applicationsURL.searchParams.set("page", `${page+1}`);
```

Reply:
Similarly for take and pendingPermits below

Reply:
updated - good call